### ISSUE TITLE
[QUESTION] Loss increase when training dinov2

### ISSUE THREAD

**Issue by YingdiZhang** [lightly-ai/lightly-train#442]
### 🤔 What’s your question?

Thanks for the great job. When training dinov2 with my own dataset, the loss curve decrease at first and increase then. I don't know if it is normal. The loss curve is as below:

<img width="1400" height="1052" alt="Image" src="https://github.com/user-attachments/assets/87fd7752-2507-4c1d-b5ea-e315afb95a35" />

### 🧠 What did you try?

Explain what you’ve already looked into or attempted.

### 📌 Related code or context (if any)

import lightly_train

if __name__ == "__main__":
  

    lightly_train.train(
        out="dinov2_ssl/vitb14_bs32_gpu8_epchauto",   
        data="/data/data_zyd/foundation_model_data",
        model="dinov2/vitb14",
        method="dinov2", 
        accelerator="gpu",
        devices=[0,1,2,3,4,5,6,7],
        # epochs=70, 
        batch_size=256, # 256 = 32 * 8 
        }
    )

```python
# Related snippet if relevant
```

**Comment by guarin** [comment]
Hi, this is normal. The DINOv2 loss increases at later stages in training before it comes down again. This is due to various hyperparameter schedules like the loss temperature that change during the training.

**Comment by YingdiZhang** [comment]
Thanks for your reply. Actually, the loss increase and don't come down later, is it normal or how I should change the hyperperrameters? The loss curve is as below:

<img width="1400" height="1052" alt="Image" src="https://github.com/user-attachments/assets/941b974a-bcbf-47d9-9142-d901ff5ac0e6" />

**Comment by guarin** [comment]
The loss seems indeed too high, you can check the plots at the bottom of our blogpost for example curves: https://www.lightly.ai/blog/dinov2-reproduction

I would try:
* Larger batch size if possible (make sure `precision=bf16`)
* Smaller learning rate

How do your images look like?

**Comment by YingdiZhang** [comment]
Thanks for your reply. My images are infrared images and remote sensing images, which is like:

![Image](https://github.com/user-attachments/assets/7b5e1cf7-0644-4dfb-a971-d1eff114170f)

![Image](https://github.com/user-attachments/assets/cdae1f35-a3d5-49aa-9fd1-93fd062b23be)

![Image](https://github.com/user-attachments/assets/d0e2f1f1-c7c6-4e67-81ed-0dbacf5b01c3)

<img width="640" height="512" alt="Image" src="https://github.com/user-attachments/assets/624e77aa-0d4d-4e59-9cc4-5b2b5584bf76" />

**Comment by guarin** [comment]
The images are diverse enough that DINOv2 pretraining should work well. The larger batch size and lower learning rate might help. What is your dataset size?

**Comment by YingdiZhang** [comment]
Thanks for your reply. My dataset size is 2M
